### PR TITLE
task/MM5-6599: Admin UI for asset relation types

### PR DESCRIPTION
**Pull request by IgorDemo**
https://digizuite.atlassian.net/browse/MM5-6599

### CODE REVIEW

**Comment by bbedigi** [comment]
@TobiasWolters do you want to add some descriptive tooltips the various inputs? 
Igor has added tooltip, but they are repeating the header at the moment.
Screenshot 2024-12-04 at 16 15 25

**Comment by TobiasWolters** [comment]
@bbedigi Sure. Most of the descriptions can be taken directly from our API documentation 😉 
- `Block publishing`: If enabled, the target asset of this relation can not be published to any channels at all.
- `Inherit channel folders`: If enabled, the target asset of this relation will inherit the channel folders of the source asset.
- Read/write: Controls what parts of item security the target asset of this relation inherits from the source asset.
- `Inherit soft delete`: If enabled, the target asset of this relation will automatically be soft deleted when the source asset is soft deleted.
- `Inherit hard delete`: If enabled, the target asset of this relation will automatically be hard deleted when the source asset is hard deleted. 
- `Available during transcode`: If enabled, the target asset of this relation will be made available when creating renditions of the source asset. Note that this is only required and supported by some transcoders.
- `Override format purposes`: The different format purposes where the source asset should use the rendition of the target asset.

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 1]
```diff
@@ -12161,3 +12161,979 @@ resource configservice_label administration_tools_search_placeholder {
...
+  key = 'ADMINISTRATION_TOOLS_SYSTEM_ASSET_RELATION_TYPES_OVERRIDE_FORMAT_PURPOSES'
+  group = 'administration-tools - system - asset-relation-types'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Override format purposes'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Tilsidesæt formatformål'
```
Comment (Poiar) [inline MM/6.2.0/config/media_manager_5/labels/Administration.dcl:12863]:
```suggestion
      default_translation = 'Overskriv format-formål'
```

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 2]
```diff
@@ -12161,3 +12161,979 @@ resource configservice_label administration_tools_search_placeholder {
...
+  key = 'ADMINISTRATION_TOOLS_SYSTEM_ASSET_RELATION_TYPES_OVERRIDE_FORMAT_PURPOSES_TOOLTIP'
+  group = 'administration-tools - system - asset-relation-types'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'The different format purposes where the source asset should use the rendition of the target asset'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'De forskellige formatformål, hvor kildeaktivet skal bruge gengivelsen af target-asset'
```
Comment (Poiar) [inline MM/6.2.0/config/media_manager_5/labels/Administration.dcl:12879]:
```suggestion
      default_translation = 'De forskellige formatformål, hvor kilde-assettet skal bruge renditionen fra mål-assetet'
```

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 3]
```diff
@@ -12161,3 +12161,979 @@ resource configservice_label administration_tools_search_placeholder {
...
+  key = 'ADMINISTRATION_TOOLS_SYSTEM_ASSET_RELATION_TYPES_DELETE_ASSET_RELATION_TYPE_ERROR_BODY'
+  group = 'administration-tools - system - asset-relation-types'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Asset relation type not deleted'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Asset-relationstype blev ikke slettet'
```
Comment (Poiar) [inline MM/6.2.0/config/media_manager_5/labels/Administration.dcl:13039]:
```suggestion
      default_translation = 'Asset-relationstypen blev ikke slettet'
```

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 4]
```diff
@@ -12161,3 +12161,979 @@ resource configservice_label administration_tools_search_placeholder {
...
+  key = 'ADMINISTRATION_TOOLS_SYSTEM_ASSET_RELATION_TYPES_CHANGES_SAVED_SUCCESS'
+  group = 'administration-tools - system - asset-relation-types'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Asset relation type saved'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Asset-relationstype blev gemt'
```
Comment (Poiar) [inline MM/6.2.0/config/media_manager_5/labels/Administration.dcl:13071]:
```suggestion
      default_translation = 'Asset-relationstypen blev gemt'
```

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 5]
```diff
@@ -12161,3 +12161,979 @@ resource configservice_label administration_tools_search_placeholder {
...
+  key = 'ADMINISTRATION_TOOLS_SYSTEM_ASSET_RELATION_TYPES_CHANGES_SAVED_ERROR_BODY'
+  group = 'administration-tools - system - asset-relation-types'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Asset relation type not saved'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Asset-relationstype blev ikke gemt'
```
Comment (Poiar) [inline MM/6.2.0/config/media_manager_5/labels/Administration.dcl:13087]:
```suggestion
      default_translation = 'Asset-relationstypen blev ikke gemt'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 6]
```diff
@@ -12161,3 +12161,979 @@ resource configservice_label administration_tools_search_placeholder {
...
+  ]
+}
+
+resource configservice_label administration_tools_system_asset_relation_types_primary_to_secondary_label {
+  key = 'ADMINISTRATION_TOOLS_SYSTEM_ASSET_RELATION_TYPES_PRIMARY_TO_SECONDARY_LABEL'
+  group = 'administration-tools - system - asset-relation-types'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Primary to secondary label'
```
[the remark this answers is not in the document]
Comment (bbedigi) [reply MM/6.2.0/config/media_manager_5/labels/Administration.dcl:13131]: How about 'Label on source asset'

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 7]
```diff
@@ -12161,3 +12161,979 @@ resource configservice_label administration_tools_search_placeholder {
...
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Primær til sekundær etiket'
```
[the remark this answers is not in the document]
Comment (bbedigi) [reply MM/6.2.0/config/media_manager_5/labels/Administration.dcl:13135]: 'Label on target asset'

**Review by Poiar — CHANGES_REQUESTED** [review]
Added new suggestions

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 8]
```diff
@@ -12161,3 +12161,995 @@ resource configservice_label administration_tools_search_placeholder {
...
+  ]
+}
+
+resource configservice_label administration_tools_system_asset_relation_types_secondary_to_primary_label {
+  key = 'ADMINISTRATION_TOOLS_SYSTEM_ASSET_RELATION_TYPES_SECONDARY_TO_PRIMARY_LABEL'
+  group = 'administration-tools - system - asset-relation-types'
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Target →  source label'
```
Comment (Poiar) [inline MM/6.2.0/config/media_manager_5/labels/Administration.dcl:13147]:
```suggestion
      default_translation = 'Target → source label'
```

File: MM/6.2.0/config/media_manager_5/labels/Administration.dcl [hunk 9]
```diff
@@ -12161,3 +12161,995 @@ resource configservice_label administration_tools_search_placeholder {
...
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Source → target label'
```
Comment (Poiar) [inline MM/6.2.0/config/media_manager_5/labels/Administration.dcl:13151]:
```suggestion
      default_translation = 'Mål → kilde label'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

**Review by Poiar — APPROVED** [review]

**Pull request merged**